app.json

```diff
@@ -2,9 +2,20 @@
   "name": "api.mozillascience.org",
   "description": "API server for science.mozilla.org",
   "addons": [
-    "heroku-postgresql"
+    "heroku-postgresql:hobby-dev"
```

Comment:
Not at all important, it's simply there so we have something spun up to test pull requests.